index.template.html

```diff
@@ -4855,91 +4856,460 @@ <h3>Extension and Import</h3>
+        {
+            "@context": "http://www.w3.org/ns/td",
+            "@type": "tm:ThingModel",
+            "title": "Ventilator Thing Model",
+            "version": {
+                "model": "1.0.0"
+            },
+            "properties": {
+                "switch": {
+                    "type": "boolean",
+                    "description": "True=On; False=Off"
+                },
+                "adjustRpm": {
+                    "type": "number",
+                    "minimum": 200,
+                    "maximum": 1200
+                }
+            }
+        }
+       </pre>
+       <pre class="LED exampleTab1">
+        {
+            "@context": "http://www.w3.org/ns/td",
+            "@type": "tm:ThingModel",
+            "title": "LED Thing Model",
+            "version": {
+                "model": "1.0.0"
+            },
+            "properties": {
+                "R": {
+                    "type": "number",
+                    "description": "Red color"
+                },
+                "G": {
+                    "type": "number",
+                    "description": "Green color"
+                },
+                "B": {
+                    "type": "number",
+                    "description": "Blue color"
+                }
+            },
+            "actions": {
+                "fadeIn": {
+                    "title": "fadeIn",
+                    "input": {
+                        "type": "number",
+                        "description": "fadeIn in ms"
+                    }
+                },
+                "fadeOut": {
+                    "title": "fadeIn",
+                    "input": {
+                        "type": "number",
+                        "description": "fadeOut in ms"
+                    }
+                }
+            }
+        }
+       </pre>
+     </aside>
 
-<section id="thing-model-td-placeholder" class="normative">
-    <h3>Placeholder</h3>
 
 <p>
-    <a>Thing Model</a> can specify which terms should be taken in a TD instance, but their values are unspecific and are first known during TD instantiation. 
-    In such a case the placeholder labeling MAY be used in Thing Model that MUST be substituted with a concrete value when TD instance is created 
-    from the Thing Model. 
-    <span class="rfc2119-assertion" id="tm-placeholder">
-        The string-based pattern of the placeholder MUST follow <code>{{\PLACEHOLDER_IDENTIFIER}}</code>, 
-        which should contain a placeholder identifier name. The identifier name can be used to identify the placeholder for the substitution process.
+    Such composition can be introduced by the usage of the <code>links</code> container. 
+    <span class="rfc2119-assertion" id="tm-compose-submodel">
+        If it is desired to provide information that a <a>Thing Model</a> consists of one or more (sub-)<a>Thing Models</a>, the <code>links</code> entries MUST use the <code>"rel":"tm:submodel"</code> 
```

Comment:
this is missing from the second table at https://pr-preview.s3.amazonaws.com/w3c/wot-thing-description/pull/1207.html#link